_posts/2013-04-11-testing.md

```diff
@@ -60,20 +60,20 @@ import { test } from 'ember-qunit';
 import startApp from '../helpers/start-app';
 var App;
 
-module('An Integration test', {
-  setup: function() {
+Qunit.module('An Integration test', {
```

Comment:
I'm not familiar how to do this. I'm guessing that this shim should be added to `ember-qunit`? What would be the next step here?